src/core_deku/ticket_table.mli

```diff
@@ -0,0 +1,34 @@
+type t [@@deriving yojson]
+
+val empty : t
+
+val balance : t -> sender:Address.t -> ticket:Ticket_id.t -> Amount.t option
+
+val deposit :
+  t -> destination:Address.t -> ticket:Ticket_id.t -> amount:Amount.t -> t
+
+val transfer :
+  t ->
+  sender:Address.t ->
+  destination:Address.t ->
+  ticket:Ticket_id.t ->
+  amount:Amount.t ->
+  (t, [> `Insufficient_funds]) result
+
+val withdraw :
+  t ->
+  sender:Address.t ->
+  ticket:Ticket_id.t ->
+  amount:Amount.t ->
+  (t, [> `Insufficient_funds]) result
+
+val tickets : t -> sender:Address.t -> (Ticket_id.t * Amount.t) Seq.t * t
```

Comment:
nitpick that I won't block on: would be nice if mli function order is in same order as implementation.